### PR TITLE
Add nsxclient for e2e

### PR DESCRIPTION
**Pull request by TaoZou1**
add nsxclient which could list nsx resources
nsxclient need the operator config file , the default is /etc/nsx-operator/nsxop.ini

### CODE REVIEW

**Review by zhengxiexie — COMMENTED** [review]
BTW, could you supplement nsxClient query checking to nsx_security_policy_test.go ?

File: test/e2e/nsxclient.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Copyright © 2019-2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: BSD-2-Clause
+
+// Auto generated code. DO NOT EDIT.
+
+package e2e
+
+import (
+	"fmt"
```
[the remark this answers is not in the document]
Comment (TaoZou1) [reply test/e2e/nsxclient.go:7]: done

File: test/e2e/nsx_security_policy_test.go [hunk 2]
```diff
@@ -35,4 +38,20 @@ func TestSecurityPolicyBasicTraffic(t *testing.T) {
 
 	err = testData.runPingCommandFromTestPod(ns, "busybox", iPs, 4)
 	assert.NotNilf(t, err, "Error when running ping command from test Pod 'busybox'")
+
+	tagScopeClusterKey := strings.Replace(common.TagScopeNamespace, "/", "\\/", -1)
```
Comment (zhengxiexie) [inline test/e2e/nsx_security_policy_test.go:42]: Perhaps it's better to encapsulate those lines into a function, e.g. 

testData.assertSecurityPolicyStatus(ns, spName, exist)
testData.assertRuleStatus(ns, ruleName, notExist)

Then other test cases could simply use these two functions to assert status before creating securitypolicy and after deleting it.

Comment (TaoZou1) [reply test/e2e/nsx_security_policy_test.go:42]: it's out the scope of this patch

**Review by zhengxiexie — APPROVED** [review]

**Pull request merged**